backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/docs/generics/report_bpblock.jrxml

```diff
@@ -31,11 +32,11 @@ FROM 	de_metas_endcustomer_fresh_reports.Docs_Generics_BPartner_Report( $P{org_i
 		<band splitType="Stretch"/>
 	</background>
 	<title>
-		<band height="135" splitType="Stretch">
+		<band height="96" splitType="Stretch">
```

Comment:
Do not change this subreport.
I will affect all docs. You shall do adjustments from your own report.